### PR TITLE
Fix uvc final

### PR DESCRIPTION
**Pull request by dNechita**

### CODE REVIEW

File: sdk/src/cameras/itof-camera/camera_itof.cpp [hunk 1]
```diff
@@ -44,6 +44,8 @@
 #include "tofi/tofi_utils.h"
 #include "tofi/tofi_config.h"
 
+#define CONFIG_DIR_NAME "config"
```
Comment (dNechita) [inline sdk/src/cameras/itof-camera/camera_itof.cpp:47]: This is defined in top level cmake file. Why is this redefined here? @SeptimiuVana

File: sdk/src/cameras/itof-camera/camera_itof.cpp [hunk 2]
```diff
@@ -277,6 +278,14 @@ aditof::Status CameraItof::setFrameType(const std::string &frameType) {
         LOG(WARNING) << "Failed to set frame type";
         return status;
     }
+
+    if ((frameType == "pcm") || (frameType == "pcm8")) {
```
Comment (dNechita) [inline sdk/src/cameras/itof-camera/camera_itof.cpp:282]: Why is this required? Isn't the depth compute required to transform from IR to AB(active brightness)? @btogorean @SeptimiuVana

File: sdk/src/cameras/itof-camera/camera_itof.cpp (hunk 1 above)
Comment (SeptimiuVana) [reply sdk/src/cameras/itof-camera/camera_itof.cpp:47]: It was not defined on master, should have deleted it at rebase.

File: sdk/src/cameras/itof-camera/camera_itof.cpp (hunk 2 above)
Comment (SeptimiuVana) [reply sdk/src/cameras/itof-camera/camera_itof.cpp:282]: Fixed.

File: sdk/src/cameras/itof-camera/camera_itof.cpp (hunk 1 above)
Comment (SeptimiuVana) [reply sdk/src/cameras/itof-camera/camera_itof.cpp]: Fixed.

**Review by SeptimiuVana — APPROVED** [review]
